datasets/huggingface.py

```diff
@@ -8,6 +8,8 @@
 
 from i6_core.util import instanciate_delayed
 
+from datasets import load_dataset, Split
```

Comment:
I would not do the import here. It should only be done inside the job.

Reply:
It needs to be here to be able to define the type hint, right?